app/views/tracks/show/unjoined.html.haml

```diff
@@ -20,55 +20,18 @@
     .md-container.container
       %h2 About #{@track.title}
       .tw-flex.tw-items-start
-        .info
-          %p C# is a multi-paradigm, statically-typed programming language with object-oriented, declarative, functional, generic, lazy, integrated querying features and type inference.
-          %p Statically-typed means that identifiers have a type set at compile time--like those in Java, C++ or Haskell--instead of holding data of any type like those in Python, Ruby or JavaScript.
+        .info= raw ParseMarkdown.(@track.about)
         %code
-          %pre
-            :preserve
-              class Card
-              {
-                  static string NewYear(int year = 2020, string sender = "me")
-                  {
-                      return $"Happy {year} from {sender}!";
-                  }
-              }
-
-              Card.NewYear();  // => "Happy 2020 from me!"
-              Card.Card(1999); // => "Happy 1999 from me!"
-              Card.Card(sender: "mom"); // => "Happy 2020 from mom!"
-
-  %section.features-section
-    .md-container.container
-      %h2 Key Features of #{@track.title}
-      .features
-        .feature
-          = graphical_icon :"features-oop"
-          %h3 Object-orientated
-          %p Quisque faucibus metus eget nulla venenatis, vel ultrices mi vehicula. Sed at volutpat odio.
-
-        .feature
-          = graphical_icon :"features-strongly-typed"
-          %h3 Strongly-typed
-          %p Quisque faucibus metus eget nulla venenatis, vel ultrices mi vehicula. Sed at volutpat odio.
-
-        .feature
-          = graphical_icon :"features-functional"
-          %h3 Functional
-          %p Quisque faucibus metus eget nulla venenatis, vel ultrices mi vehicula. Sed at volutpat odio.
-
-        .feature
-          = graphical_icon :"features-lazy"
-          %h3 Lazy
-          %p Quisque faucibus metus eget nulla venenatis, vel ultrices mi vehicula. Sed at volutpat odio.
-
-        .feature
-          = graphical_icon :"features-declarative"
-          %h3 Declarative
-          %p Quisque faucibus metus eget nulla venenatis, vel ultrices mi vehicula. Sed at volutpat odio.
-
-        .feature
-          = graphical_icon :"features-generic"
-          %h3 Generic
-          %p Quisque faucibus metus eget nulla venenatis, vel ultrices mi vehicula. Sed at volutpat odio.
-
+          %pre= @track.snippet
+
+  - if @track.key_features.present?
+    %section.features-section
+      .md-container.container
+        %h2 Key Features of #{@track.title}
+
+        .features
+          - @track.key_features.first(6).each do |key_feature|
```

Comment:
Can we extract a constant for `6` instead of using it directly? I'd be happy to just use a view partial and declare the constant on the top of that partial for now.

Reply:
I've extracted to a view partial: https://github.com/exercism/v3-website/pull/386

The constant 6 is used in other places too, so that's probably a bigger discussion :)